packages/amplify-graphql-types-generator/src/loading.ts

```diff
@@ -62,6 +64,13 @@ export function loadAndMergeQueryDocuments(inputPaths: string[], tagName: string
     })
     .filter((source): source is Source => Boolean(source));
 
+    if (originalPlatform === 'win32') {
```

Comment:
If going with the above suggestion

```suggestion
    if (process.platform === 'win32') {
```